tests/test_tree_sitter.py

```diff
@@ -1,16 +1,28 @@
 # pylint: disable=missing-docstring
 
+import os
 import re
 from unittest import TestCase
 from os import path
 from tree_sitter import Language, Parser
 
-LIB_PATH = path.join("build", "languages.so")
+# cibuildwheel uses a funny working directory when running tests.
+# This is by design, this way tests import whatever is installed and not from the project.
+# This means that we can't load anything relative to the current working directory.
+project_root = path.dirname(path.dirname(path.abspath(__file__)))
+
+LIB_PATH = path.join(project_root, "build", "languages.so")
+if os.getenv("PY_TREE_SITTER_TESTS_FORCE_REBUILD_LANGUAGES"):
+    try:
+        os.remove(LIB_PATH)
+    except FileNotFoundError:
+        pass
+
```

Comment:
Would it be worth it to have a .so name that contain the python version/platform in the same way that pyc files ends with for example `cpython-310` ? That would avoid any conflicts.

Reply:
It actually has to be something like `cpython-310-amd64`, because 32-bit and 64-bit wheels are built in the same environment. I'll change this.

Reply:
Sure, and like also sys.platform to be sure ?

Reply:
Turns out that just placing the languages binary to current working directory works. This seems to be how cibuildwheel is meant to be used: it makes a new working directory for each wheel build just so that you can make a mess in it to be cleaned up before the next build.